support/client/lib/vwf/model/kineticjs.js

```diff
@@ -521,9 +534,7 @@ define( [ "module",
 
                         case "stroke":
                             var vwfColor = new utility.color( propertyValue );
-                            if ( vwfColor ) {
```

Comment:
Is there a particular reason the if statement was removed?  I'm not sure if it ever returns a null or undefined vwfColor but if it does we'll get an error on toString without the if statement.  If we want to clear the stroke when we see an undefined vwfColor, I think we'd have to pass null